samza-rest/src/main/java/org/apache/samza/monitor/SamzaMonitorService.java

```diff
@@ -40,6 +41,7 @@
 public class SamzaMonitorService {
 
     private static final Logger LOGGER = LoggerFactory.getLogger(SamzaMonitorService.class);
+    private static final SecureRandom RAND = new SecureRandom();
```

Comment:
Nitpick: RANDOM

Reply:
Changed the constant from RAND to RANDOM.